src/main/java/oakbot/listener/DadJokeListener.java

```diff
@@ -23,6 +23,7 @@
 
 	private final Pattern regex = Pattern.compile( //@formatter:off
 		"(?i)" +
+		"(?:^|@[^ ]+|:\\d+|[.?!])\\s*" +
```

Comment:
I wonder if `\b` should be used instead of `\s*`? I have been using `\b` in the rest of the regex. What do you think?

Reply:
`\s*` is actually necessary here. Ref. [this regex101](https://regex101.com/r/ah0alQ/1) - if we just do word bounds, it'll match `:1234<eow>` and fail to match anything else, because the space between :1234 and `I'm <something>` means the "I'm ..:" isn't detected

Reply:
Huh, weird.